sites/public/src/pages/applications/start/choose-language.tsx

```diff
@@ -33,13 +33,14 @@ const loadListing = async (
   stateFunction,
   conductor,
   context,
-  language
+  language,
+  isPreview
 ) => {
   const response = await axios.get(`${backendApiBase}/listings/${listingId}`, {
     headers: { language },
   })
   conductor.listing = response.data
-  const applicationConfig = retrieveApplicationConfig(conductor.listing) // TODO: load from backend
+  const applicationConfig = retrieveApplicationConfig(conductor.listing, isPreview) // TODO: load from backend
```

Comment:
Are we loading from backend yet?

Reply:
Don't believe so! That comment has been there for 4 years though so unsure if that's still the goal